docs/docs/Entendimento do problema/rf_rnf.md

```diff
@@ -1,7 +1,564 @@
+
+
+
+### RF-33: Verificação de Vencimento de Boletos
+O sistema deve ser capaz de verificar automaticamente o vencimento dos boletos de pagamento gerados na plataforma.
+
+**Casos de Teste**
+
+| ID    | Descrição                                                                                                   |
+| ----- | ----------------------------------------------------------------------------------------------------------- |
+| CT-121| Verificar se o sistema pode verificar o vencimento dos boletos automaticamente                               |
+| CT-122| Verificar se o sistema envia notificações quando um boleto está próximo do vencimento                        |
+| CT-123| Verificar se uma mensagem de erro é exibida quando ocorre falha na verificação do vencimento do boleto        |
+
+
+
+### RF-34: Envio de Notificação de Vencimento por E-mail
+O sistema deve ser capaz de enviar notificações por e-mail aos usuários quando contratos ou boletos estão próximos do vencimento.
+
+**Casos de Teste**
+
+| ID    | Descrição                                                                                                   |
+| ----- | ----------------------------------------------------------------------------------------------------------- |
+| CT-124| Verificar se o sistema pode enviar notificações por e-mail para contratos próximos do vencimento             |
+| CT-125| Verificar se o sistema pode enviar notificações por e-mail para boletos próximos do vencimento               |
+| CT-126| Verificar se uma mensagem de confirmação é exibida após o envio da notificação por e-mail                    |
+| CT-127| Verificar se uma mensagem de erro é exibida quando ocorre falha no envio da notificação por e-mail            |
+
+
+
+### RF-35: Envio de Notificação de Boleto Gerado por E-mail
+O sistema deve ser capaz de enviar notificações por e-mail aos usuários quando novos boletos de pagamento são gerados.
+
+**Casos de Teste**
+
+| ID    | Descrição                                                                                                   |
+| ----- | ----------------------------------------------------------------------------------------------------------- |
+| CT-128| Verificar se o sistema pode enviar notificações por e-mail quando novos boletos são gerados                  |
+| CT-129| Verificar se o sistema envia o boleto gerado como anexo no e-mail                                            |
+| CT-130| Verificar se uma mensagem de confirmação é exibida após o envio da notificação por e-mail                    |
+| CT-131| Verificar se uma mensagem de erro é exibida quando ocorre falha no envio da notificação por e-mail            |
+
+
+
+### RF-36: Envio de Notificação de Contrato Assinado por E-mail
+O sistema deve ser capaz de enviar notificações por e-mail aos usuários quando novos contratos são assinados digitalmente.
```

Comment:
Acredito que esse requisito deve ser modificado para: O sistema deve ser capaz de enviar notificações por e-mail aos usuários quando este for adicionado na plataforma, mencionando o contrato de locação em questão.